.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -185,6 +204,7 @@ stages:
       # get load test status and wait till it's done
       - task: AzureCLI@2
         displayName: 'Wait for load test to be finished'
+        continueOnError: true # shows a warning but continues the pipeline
```

Comment:
is that for the failed baseline comparison? maybe add that. But I guess it also means that the pipeline would continue if something else breaks?!

Reply:
yes, that is correct but i've found no other option to visualize it in the pipeline

this is the same approach we've used for locust - and yes it would also continue if something else breaks